#! /bin/sh /usr/share/dpatch/dpatch-run
## 026_omit_extra_libs.dpatch by Stefan Fritsch <sf@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: #463399

@DPATCH@
diff -urNad trunk~/apr-config.in trunk/apr-config.in
--- trunk~/apr-config.in	2009-06-07 21:08:49.000000000 +0200
+++ trunk/apr-config.in	2009-06-07 21:10:50.573842471 +0200
@@ -36,7 +36,7 @@
 CPPFLAGS="@EXTRA_CPPFLAGS@"
 CFLAGS="@EXTRA_CFLAGS@"
 LDFLAGS="@EXTRA_LDFLAGS@"
-LIBS="@EXTRA_LIBS@"
+LIBS=""
 EXTRA_INCLUDES="@EXTRA_INCLUDES@"
 SHLIBPATH_VAR="@shlibpath_var@"
 APR_SOURCE_DIR="@apr_srcdir@"
